ui/component/or-dashboard-builder/src/widgets/or-image-widget.ts

```diff
@@ -0,0 +1,366 @@
+            });
+            return assets;
+        }
+    }
+}
+
+@customElement("or-image-widgetsettings")
+export class OrImageWidgetSettings extends LitElement {
+    static get styles() {
+        return [style, widgetSettingsStyling];
+    }
+
+    private _fileElem!: HTMLInputElement;
+    private expandedPanels: string[] = [i18next.t('attributes'), i18next.t('marker coordinates'), i18next.t('image settings')];
+    private loadedAssets?: Asset[];
+    @property()
+    public readonly widget?: DashboardWidget;
+
+    // same implementation as /or-components/src/or-file-uploader.ts
+    // file explorer dialogue opens with supported file types
+    @property({ attribute: false })
+    public accept: string = "image/png,image/jpeg,image/vnd.microsoft.icon,image/svg+xml";
+    // UI Rendering
+    render() {
+        const config = JSON.parse(JSON.stringify(this.widget!.widgetConfig)) as ImageWidgetConfig; // duplicate to edit, to prevent parent updates. Please trigger updateConfig()
+
+        var output = html`
+            <div>
+                ${this.generateExpandableHeader(i18next.t('attributes'))}
+            </div>
+            <div>
+                ${this.expandedPanels.includes(i18next.t('attributes')) ? html`
+                    <or-dashboard-settingspanel .type="${SettingsPanelType.MULTI_ATTRIBUTE}" .widgetConfig="${this.widget!.widgetConfig}"
+                    @updated="${(event: CustomEvent) => {
+                        this.onAttributesUpdate(event.detail.changes);
+                        this.updateConfig(this.widget!, event.detail.changes.get('config'));
+                }}"
+                    ></or-dashboard-settingspanel>
+                ` : null}
+        </div>
+        <div>
+            ${this.generateExpandableHeader(i18next.t('marker coordinates'))}
+        </div>
+        <div style="display: flex; justify-content: start; flex-direction: row; flex-wrap: wrap; align-items: flex-start; min-width: 150px;">
+            ${ this.expandedPanels.includes(i18next.t('marker coordinates')) ? this.prepareCoordinateEntries(config, i18next.t('marker coordinates')): null}
+        </div>
+            <div>
+                ${this.generateExpandableHeader(i18next.t('image settings'))}
+            </div>
+            <div>
+                ${this.expandedPanels.includes(i18next.t('image settings')) ? html`
+                    <div style="padding: 24px 24px 48px 24px;">
+                        <div>
+                        <input type="file" @change="${this.handleFileInputChange}" accept="${this.accept}"/> 
+                        </div>
+                    </div>
+                ` : null}
+            </div>
+        `
+        return output;
+    }
+
+    private prepareCoordinateEntries(config: ImageWidgetConfig, name: string){
+        var min = 0;
+        var max = 100;
+        if (config.attributeRefs && config.attributeRefs.length > 0) {
+            this.updateConfig(this.widget!, config);
```

Comment:
Should this be here? Since `prepareCoordinateEntries()` is called during every render / UI update, it will update the config each time as well.